lib/panel/dock.js

```diff
@@ -14,6 +10,11 @@ class PanelDock {
   constructor(delegate: Object) {
     this.element = document.createElement('div')
     this.subscriptions = new CompositeDisposable()
+
+    const React = require('react')
```

Comment:
Although `require` caching _works_, it's probably better to not rely on it exclusively and make the references to these at the file level.